packages/firebase_core/firebase_core/example/web/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <title>Firebase Core Example</title>
+</head>
+<body>
+  <script src="https://www.gstatic.com/firebasejs/7.5.0/firebase-app.js"></script>
+  <script src="main.dart.js" type="application/javascript"></script>
```

Comment:
Is this really enough to get the demo going on web? No app IDs, or anything like that?

Reply:
The app is initialized in `lib/main.dart` so there is no need to put the default config in the HTML here